src/score_hv/stats_utils.py

```diff
@@ -0,0 +1,105 @@
+"""
+Copyright 2022 NOAA
+All rights reserved.
+
+Collection of methods to calculate statistics requested
```

Comment:
I think we could have a more robust description here. Additionally, I don't believe that it's correct to say this file is to support the tests. It's to support the main functionality of the program, which also gets tested, so that could be confusing to future users.